examples/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_executable(run_ddx ./run_ddx.c)
```

Comment:
In case of a subproject, such a target name "run_ddx" may be used by the superproject.

Reply:
Ok, so what do you suggest?